parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -196,36 +201,30 @@ Expand-Archive scripts.zip -DestinationPath "C:\\AzureData\\"
 . c:\AzureData\windows\windowscsehelper.ps1
 # util functions only can be used after this line, for example, Write-Log
 
-# Download CSE function scripts
-Write-Log "Getting CSE scripts"
-$tempfile = 'c:\csescripts.zip'
-DownloadFileOverHttp -Url $global:CSEScriptsPackageUrl -DestinationPath $tempfile
-Expand-Archive $tempfile -DestinationPath "C:\\AzureData\\windows"
-Remove-Item -Path $tempfile -Force
-
-# Dot-source cse scripts with functions that are called in this script
-. c:\AzureData\windows\azurecnifunc.ps1
-. c:\AzureData\windows\calicofunc.ps1
-. c:\AzureData\windows\configfunc.ps1
-. c:\AzureData\windows\containerdfunc.ps1
-. c:\AzureData\windows\kubeletfunc.ps1
-. c:\AzureData\windows\kubernetesfunc.ps1
-
-$useContainerD = ($global:ContainerRuntime -eq "containerd")
-$global:KubeClusterConfigPath = "c:\k\kubeclusterconfig.json"
-$fipsEnabled = [System.Convert]::ToBoolean("{{ FIPSEnabled }}")
-$windowsSecureTlsEnabled = [System.Convert]::ToBoolean("{{GetVariable "windowsSecureTlsEnabled" }}");
-
 try
 {
+    Write-Log ".\CustomDataSetupScript.ps1 -MasterIP $MasterIP -KubeDnsServiceIp $KubeDnsServiceIp -MasterFQDNPrefix $MasterFQDNPrefix -Location $Location -AADClientId $AADClientId -NetworkAPIVersion $NetworkAPIVersion -TargetEnvironment $TargetEnvironment"
+
+    # Download CSE function scripts
+    Write-Log "Getting CSE scripts"
+    $tempfile = 'c:\csescripts.zip'
+    DownloadFileOverHttp -Url $global:CSEScriptsPackageUrl -DestinationPath $tempfile

```

Comment:
I think we should add logs for the retry in the
 try {
            $args = @{Uri=$Url; Method="Get"; OutFile=$DestinationPath}
            Retry-Command -Command "Invoke-RestMethod" -Args $args -Retries 5 -RetryDelaySeconds 10
        } catch {
            Set-ExitCode -ExitCode $global:WINDOWS_CSE_ERROR_DOWNLOAD_FILE_WITH_RETRY -ErrorMessage "Failed in downloading $Url. Error: $_"
        }

so we know whether the retry works as expected.

Reply:
Good suggestion. I can add logs for the retry in `function Retry-Command `